lib/rnaget/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+services:
+  rnaget:
+    build:
+      context: $PWD/lib/rnaget/rnaget
+    image: ghcr.io/bento-platform/transcriptomics_data_service:edge
+    labels:
+      - "candigv2=rnaget"
+    logging:
+      driver: "fluentd"
+      options:
+        tag: "{{.Name}}"
+    ports:
+      - "${RNAGET_PORT}:5000"
+    environment:
+      - CORS_ORIGINS="*"
+      - BENTO_AUTHZ_SERVICE_URL=""
+      - OPA_URL=${OPA_PRIVATE_URL}
+      - VAULT_URL=${VAULT_PRIVATE_URL}
+      - KEYCLOAK_PUBLIC_URL=${KEYCLOAK_PUBLIC_URL}
```

Comment:
Are these two needed? I see OPA_URL is needed because of the `authx.auth.verify_service_token` call, but the other two don't look like they're used anywhere. Is this future-proofing in case we need more auth?

Reply:
Yeah, good catch. I originally had it there because I needed to use other endpoints as well. I can remove it.